cmake/CMakeLists.txt

```diff
@@ -1,7 +1,13 @@
-cmake_minimum_required(VERSION 3.29)
+cmake_minimum_required(VERSION 3.28)
```

Comment:
why downgrade cmake ?

Reply:
Like said IRL, my pc isn't able to update for some unknow reason to this version
Fedora packages must haven't release it yet ?